cluster-density.go

```diff
@@ -61,6 +62,7 @@ func NewClusterDensity(wh *workloads.WorkloadHelper, variant string) *cobra.Comm
 	cmd.Flags().DurationVar(&podReadyThreshold, "pod-ready-threshold", 2*time.Minute, "Pod ready timeout threshold")
 	cmd.Flags().IntVar(&iterations, "iterations", 0, fmt.Sprintf("%v iterations", variant))
 	cmd.Flags().BoolVar(&churn, "churn", true, "Enable churning")
+	cmd.Flags().IntVar(&churnCycles, "churn-cycles", 3, "Churn cycles to execute")
```

Comment:
We should use 0 by default, to prevent enabling this feature accidentally, we can specify cycles directly in prow

Reply:
Updated as suggested.